azure-pipelines.yml

```diff
@@ -9,6 +9,8 @@ trigger:
 jobs:
 - job: Linux
 
+  timeoutInMinutes: 0
+
```

Comment:
what does this do?

Reply:
I believe it allows the azure jobs to run the maximum time (more than the default of 60 minutes).

Reply:
That doesn't appear to be documented. Maybe just set it to the maximum (360 minutes)?

Reply:
Quoting from https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml&viewFallbackFrom=vsts#timeouts

"The timeoutInMinutes allows a limit to be set for the job execution time. When not specified, the default is 60 minutes. When 0 is specified, the maximum limit is used (described above)."

Reply:
Ah, I missed that. Thanks.